controllers/flowcollector_controller.go

```diff
@@ -231,10 +230,8 @@ func (r *FlowCollectorReconciler) handleNamespaceChanged(
 func (r *FlowCollectorReconciler) SetupWithManager(ctx context.Context, mgr ctrl.Manager) error {
 	builder := ctrl.NewControllerManagedBy(mgr).
 		For(&flowslatest.FlowCollector{}).
-		Owns(&corev1.ConfigMap{}).
 		Owns(&appsv1.Deployment{}).
 		Owns(&appsv1.DaemonSet{}).
-		Owns(&ascv2.HorizontalPodAutoscaler{}).
```

Comment:
My understanding of removing this is that we will not get notification for reconciliation if the  HPA is manually modified.
https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/builder#Builder.Owns

Reply:
Yeah tbh I'm not sure why I removed HPA, my point is really about configmap, I'll put HPA back.
It's a trade-off, might be loosing some reconcile events, but the gain overcomes the disadvantages